spec/tracer_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+# typed: false
+
+describe Redcord::Tracer do
+  class FakeTracer
+    def trace(*)
+      yield
+    end
+  end
+
+  context 'when tracer is set' do
+    include Redcord::Tracer::ClassMethods
+
+    let(:tracer) { proc { FakeTracer.new } }
```

Comment:
Sorry maybe noob question, but I don't understand what this unit test is doing. Is tracer supposed to be used somewhere? e.g. `tracer.call.trace`?

Reply:
Good question! when we call `trace`, it will use `tracer.call.trace` (tracer is mocked here, normally it'd set using `.tracer { ... }` )